account-kit/smart-contracts/src/msca/client/client.test.ts

```diff
@@ -3,7 +3,7 @@ import {
   type SmartAccountSigner,
   erc7677Middleware,
 } from "@aa-sdk/core";
-import { type Address, custom, parseEther } from "viem";
+import { type Address, custom, parseEther, stringify } from "viem";
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'stringify' is defined but never used.
<!-- __reviewdog__:ChA1NjkxYzBlMWEyYzk5N2EwEgZyZGpzb24= -->